odoo_sentinel/__init__.py

```diff
@@ -904,7 +904,7 @@ def main():
         '-p', '--profile', dest='profile', default='sentinel',
         help='OdooRPC profile to use.')
     parser.add_argument(
-        '-c', '--config', dest='config_file', default='~/.odoorpcrc',
```

Comment:
Why did you change the configuration file's location ?
https://pythonhosted.org/OdooRPC/tuto_session.html?highlight=odoorpcrc